### DISCUSSION TITLE
Custom user registration page

### DISCUSSION THREAD

**Discussion by workingJoey** [Budibase/budibase#14841] [Help]
![image](https://github.com/user-attachments/assets/567a48c9-ce39-42e4-bd3d-42298a73305a)

Above is the login page for budibase. I was wondering if it is possible to modify this page to include a registration page or add a link somewhere for user to navigate to a registration page form

Im running version 2.8.2 self hosted

**Comment by mikesealey** [comment]
Hey @workingJoey - You can add new users by inviting them with their email address. Read more [here](https://docs.budibase.com/docs/adding-users)

**Reply by workingJoey** [reply]
is there no way for them to register it themselves. Maybe use an api to create public user

**Reply by ConorWebb96** [reply]
This thread may be helpful for you. https://github.com/Budibase/budibase/discussions/6098
